crates/wasmtime/src/runtime/stack.rs

```diff
@@ -1,6 +1,6 @@
-use std::{ops::Range, sync::Arc};
-
+use crate::prelude::*;
 use anyhow::Error;
+use std::{ops::Range, sync::Arc};
```

Comment:
`std`?

Reply:
Yeah any modules which are gated on the `std` feature still import from `std` and don't need to get switched over. Here this module is only used for `async` which requires the `std` feature at this time, so we'll need to change this in the future to an import from `core` if we want the `async` feature to work on no_std for example.